packages/neuron-ui/src/components/SpecialAsset/specialAsset.module.scss

```diff
@@ -91,7 +91,7 @@
       color: #ffffff80 !important;
     }
 
-    &.hasTooltip:disabled[data-tooltip] {
+    &.hasTooltip[data-tooltip] {
```

Comment:
I think If the element has `hasTooltip` classname, the tooltip should be display.But not should be display with `disabled`.

Reply:
If so, the condition to add `style.hasTooltip` should be updated too, because other assets are designed to pop the tooltip when the button is disabled.

Reply:
I review the code https://github.com/nervosnetwork/neuron/blob/35e8cf8949d33971d91b1bb98fe563e45ef21149/packages/neuron-ui/src/components/SpecialAsset/index.tsx#L169-L176. If the element has hasTooltip classname, it must have `disabled` attribution.So i think i can remove `&[disabled]`